setup.py

```diff
@@ -14,12 +14,12 @@
     namespace_packages = ['sio', 'sio.compilers', 'sio.executors'],
 
     install_requires = [
-        'filetracker>=2.1.5,<3.0',
+        'filetracker[server]==2.2.0',
```

Comment:
I suggest to use the same type of bounds as before:
`filetracker[server]>=2.2.0,<3.0`